go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/google/gofuzz v1.2.0
 	github.com/onsi/ginkgo v1.12.1
 	github.com/onsi/gomega v1.10.1
-	github.com/openshift/api v3.9.0+incompatible
+	github.com/openshift/api v0.0.0-20200116145750-0e2ff1e215dd
```

Comment:
why did that version change?

Reply:
the I ingress was not in config/v1 version 3.9

Reply:
yeah, also it broke for everyone and now they use what you did

see https://github.com/openshift/api/issues/456 to track